src/include/kdbprivate.h

```diff
@@ -324,6 +327,15 @@ struct _KeySet
 #endif
 };
 
+struct _SendNotificationHook
+{
+	struct _Plugin * plugin;
+	struct _SendNotificationHook * next;
+
+	kdbHookSendNotificationGetPtr get;
+	kdbHookSendNotificationGetPtr set;
+};
```

Comment:
```suggestion
typedef struct _SendNotificationHook
{
	struct _Plugin * plugin;
	struct _SendNotificationHook * next;

	kdbHookSendNotificationGetPtr get;
	kdbHookSendNotificationGetPtr set;
} SendNotificationHook;
```

I'd use a `typedef`, it makes using the type easier.